### PR TITLE
docs: Update localnet docs

### PR DESCRIPTION
**Pull request by abukosek**
Update docs for https://github.com/oasisprotocol/oasis-web3-gateway/pull/653.

### CODE REVIEW

File: docs/dapp/tools/localnet.mdx [hunk 1]
```diff
@@ -178,8 +180,7 @@ contract and wallet balance will be lost after you quit the Docker container!
 ## GitHub Actions
 
 You can easily integrate localnet into your CI/CD workflow. Use the example
-GitHub Action configuration to start a Sapphire or Emerald stack, fund the
-first two test accounts, and expose the necessary ports for testing.
+GitHub Action configuration to start a Sapphire or Emerald stack and expose the necessary ports for testing.
```
Comment (matevz) [inline docs/dapp/tools/localnet.mdx]: Trim to 80 chars.

File: docs/dapp/tools/localnet.mdx [hunk 2]
```diff
@@ -195,7 +196,7 @@ jobs:
           - 8545:8545
           - 8546:8546
         env:
-          OASIS_DEPOSIT_BINARY: /oasis-deposit -n 2
+          OASIS_DOCKER_START_EXPLORER: no
```
Comment (matevz) [inline docs/dapp/tools/localnet.mdx:200]: This makes me wonder, how can one provide -to and -n parameters in the CI after we migrated to the Oasis CLI?

**Review by matevz — APPROVED** [review]

File: docs/dapp/tools/localnet.mdx (hunk 1 above)
Comment (abukosek) [reply docs/dapp/tools/localnet.mdx]: Done :)

File: docs/dapp/tools/localnet.mdx (hunk 2 above)
Comment (abukosek) [reply docs/dapp/tools/localnet.mdx:200]: :man_shrugging:

**Pull request merged**